x-pack/legacy/plugins/apm/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Can we keep the behaviour we are already doing for other overrides of keep everything from eslint in a global configuration other than have nested config files? Like for example https://github.com/elastic/kibana/blob/master/.eslintrc.js#L390,

You can just add other object with those rules on that line,

Reply:
@mistic added another item to the array that only applies to `.ts(x)` files (it was already applied to `.js` files).